_data/chaselundell/chaselundell.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Chase
+institution: Utah State University
+profile_pic: usucrest.png
+quote: Programming is like writing a book... except when you miss a single comma on page 126 th whole thing makes no sense.
```

Comment:
Looks like your quote is 116 characters, including spaces. According to the boilerplate code available in README.md, this can only be up to 100 characters in length. If you could maybe shorten that quote, we'll be golden!

From README.md
> name: FULLNAME-OR-NICKNAME # No longer than 28 characters
> institution: INSTITUTION-NAME 🚩 # no longer than 58 characters
> profile_pic: mona.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
> quote: YOUR-SENIOR-QUOTE # **no longer than 100 characters**, avoid using quotes(") to guarantee the format remains the same.
> github_user: YOUR-GITHUB-USERNAME